app/web/typo3conf/ext/theme/Classes/ViewHelpers/ScssBreakpointViewHelper.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Sup7even\Theme\ViewHelpers;
+
+use \TYPO3\CMS\Fluid\Core\ViewHelper\AbstractViewHelper;
+use \TYPO3\CMS\Extbase\Utility\DebuggerUtility;
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Core\Page\PageRenderer;
+use function GuzzleHttp\json_encode;
+
+/**
+ * parses a given scss file and extracts the breakpoint variables
+ */
+class ScssBreakpointViewHelper extends AbstractViewHelper {
```

Comment:
I like the approach of having all bs4 breakpoints dynamically imported thingi. But after we talked about to not deploy scss to production, what do you think?